libs/Microsoft.MixedReality.WebRTC/PeerConnection.cs

```diff
@@ -1061,6 +1061,50 @@ public void RemoveLocalAudioTrack()
             PeerConnectionInterop.PeerConnection_RemoveLocalAudioTrack(_nativePeerhandle);
         }
 
+        class AudioReadStream : IAudioReadStream
+        {
+            IntPtr _nativeStreamHandle = IntPtr.Zero;
+            internal AudioReadStream(PeerConnectionHandle peerHandle, int bufferMs)
+            {
+                uint res = AudioReadStreamInterop.Create(peerHandle, bufferMs, out _nativeStreamHandle);
+                Utils.ThrowOnErrorCode(res);
+            }
+            ~AudioReadStream()
+            {
+                Dispose(false);
+            }
+
+            public void ReadAudio(int sampleRate, float[] data, int channels)
+            {
+                AudioReadStreamInterop.Read(_nativeStreamHandle, sampleRate, data, data.Length, channels);
+            }
+
+            public void Dispose()
+            {
+                Dispose(true);
+                GC.SuppressFinalize(this);
+            }
+            protected void Dispose(bool disposing)
+            {
+                if (this._nativeStreamHandle != IntPtr.Zero)
+                {
+                    AudioReadStreamInterop.Destroy(_nativeStreamHandle);
+                    this._nativeStreamHandle = IntPtr.Zero;
+                }
+            }
+        }
+
+        /// <summary>
+        /// High level interface for consuming WebRTC audio streams.
```

Comment:
This doesn't say much about why one would use that and what features it enables. Can we add an example of feature to show why/how to use an audio read stream?